src/runner/runner.go

```diff
@@ -240,3 +242,8 @@ func dumpMetrics(clientID string) error {
 
 	return nil
 }
+
+func resetMetrics(clientId string) {
```

Comment:
```suggestion
func resetMetrics(clientID string) {
```
As per linter suggestion